### PR TITLE
card with badge, dismiss, text overlay, textonly commit

### PR DESCRIPTION
**Pull request by mmanjuprasad**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -37,12 +37,34 @@
   .card-content:hover{
       background-color: #f8f8f8;
```
Comment (git-0r) [inline styles.css:38]: suggestion: you can create variable for background-color

File: styles.css [hunk 2]
```diff
@@ -37,12 +37,34 @@
...
+  .author-details-overlay{
+    /* padding: 1.4rem; */
+    font-weight: 500;
+    font-size: 1.2rem;
+    position: relative;
+    top: -4rem;
+    color: white;
+    /* bottom: 20px; */
+    /* left: 10px; */
+    /* z-index: -auto; */
```
Comment (git-0r) [inline styles.css:64]: here also you can use variable, and remove commented out code.

**Review by git-0r — COMMENTED** [review]
you can use variables for values which are repeating, and remove commented out code.

File: styles.css [hunk 3]
```diff
@@ -51,7 +73,7 @@
   
   }
   
-  .card-basic-w-h .content{
+  .card-basic .content{
       padding: 1.6rem 1.6rem;
```
Comment (yaswanthmyneni) [inline styles.css:77]: changes: you can remove one 1.6rem here.

File: styles.css [hunk 4]
```diff
@@ -140,6 +162,32 @@
...
     object-fit: cover;
 }
+
+/* close button on card badge styling  */
+
+.close-badge{
+  height: 2rem;
+  width: 2rem;
+  color:#f8f8f8;
+  background-color: rgb(19, 18, 18);
```
Comment (yaswanthmyneni) [inline styles.css:172]: changes: use css variables for colors.

File: styles.css [hunk 5]
```diff
@@ -140,6 +162,32 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+}
+
+.cibd{
+  font-size: 2.5rem;
+    /* bottom: 10px;
+    right: 10px; */
```
Comment (yaswanthmyneni) [inline styles.css:183]: changes: remove commented code

**Pull request merged**